internal/storage/storage.go

```diff
@@ -91,7 +91,14 @@ func (s *Storage) init() error {
 		}
 	}
 
-	return nil
+	flag, err := s.CheckCluster()
```

Comment:
CheckTypeFiledExist is better? and flag -> isExist?

Reply:
I see, I'll fix this latter.